sentry-android-ndk/src/main/java/io/sentry/android/ndk/IModuleListLoader.java

```diff
@@ -0,0 +1,10 @@
+package io.sentry.android.ndk;
+
+import io.sentry.protocol.DebugImage;
+
+/** Used only for making the Module list loader testable */
+interface IModuleListLoader {
+  DebugImage[] getModuleList();
```

Comment:
`m`: Why not rename it do `getDebugImage` or `loadDebugImage` when the return type is `DebugImage`?

Reply:
renamed, I followed the sentry-native convention `sentry_get_modules_list`, but for Java it'd make more sense `load`, agreed.